pages/docs/tutorials/mpp/multiplatform-library.md

```diff
@@ -267,6 +267,8 @@ class Base64Test {
 
 </div>
 
+Copy gradlew, gradlew.bat and gradle/wrapper/gradle-wrapper.jar from another project setup with gradle.
```

Comment:
Advising to copy something from something else as one of the first steps is not the best option for the tutorial.
It would be much better if we asked to [generate the wrapper instead](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:adding_wrapper)

Reply:
Yup, just realized. Will update the PR. Any gradle above 4.7 did it for me.

Reply:
Thanks!